change/@fluentui-react-93efd278-7a3a-41b8-aec3-e8a390aa7304.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "fix: Callback to update container scroll position on text height update",
```

This goes in the release notes for the whole package, so it's good to specify which component the change is in.
```suggestion
  "comment": "TextField: Add callback to update container scroll position on text height update",
```